src/markdown/tutorial/part-2/10-service-injection.md

```diff
@@ -309,7 +309,7 @@ We will take advantage of this capability in our component test:
    });
 ```
 
-In this component test, we *[registered][TODO: link to register]* our own router service with Ember in the `beforeEach` hook. When our component is rendered and requests the router service to be injected, it will get an instance of our `MockRouterService` instead of the built-in router service.
+In this component test, we *[registered](../../../release/applications/dependency-injection/#toc_factory-registrations)* our own router service with Ember in the `beforeEach` hook. When our component is rendered and requests the router service to be injected, it will get an instance of our `MockRouterService` instead of the built-in router service.
```

Comment:
```suggestion
In this component test, we *[registered](../../../applications/dependency-injection/#toc_factory-registrations)* our own router service with Ember in the `beforeEach` hook. When our component is rendered and requests the router service to be injected, it will get an instance of our `MockRouterService` instead of the built-in router service.
```